raiden/raiden_service.py

```diff
@@ -473,59 +483,90 @@ def get_block_number(self) -> BlockNumber:
     def on_message(self, message: Message):
         self.message_handler.on_message(self, message)
 
-    def handle_state_change(self, state_change: StateChange):
+    def handle_and_track_state_change(self, state_change: StateChange):
```

Comment:
Can you add two docstrings. One to `handle_and_track_state_change()` and one to `handle_state_change() explaining in one sentence how they can be used differently to track/handle exception as you commented here: https://github.com/raiden-network/raiden/pull/2985#issuecomment-459484805